pkg/provider/provider.go

```diff
@@ -223,6 +223,15 @@ func buildTestEnvironment() { //nolint:funlen
 	operators := createOperators(data.Csvs, data.Subscriptions, data.AllInstallPlans, data.AllCatalogSources, false, false, true)
 	env.Operators = operators
 	logrus.Infof("Operators found: %d", len(env.Operators))
+	for _, pod := range env.Pods {
+		isCreatedByDeploymentConfig, err := pod.CreatedByDeploymentConfig()
+		if err != nil {
+			logrus.Warnf("Pod %s, err: %s", pod.String(), err)
```

Comment:
Please try to be more verbose with this kind of log traces. Also, maybe using `logrus.Errorf` is more appropriate as you're actually reporting an important error here. My suggestion:
```
logrus.Errorf("Failed to know whether pod %s was created by a DeploymentConfig: %v", pod.String(), err)
```